_schemaV2.graphql

```diff
@@ -2562,13 +2564,17 @@ type AuthenticationType {
 }
 
 type Author {
+  bio(format: Format): String
+
   # A globally unique ID.
   id: ID!
+  image: Image
+  initials(length: Int = 3): String
 
   # A type-specific ID.
   internalID: ID!
   name: String
-  profileHandle: String
```

Comment:
`profileHandle` doesn't show up anywhere in Positron. It's not used in Force either. `twitterHandle` is an actual field, however.